vcs/vcs2vtk.py

```diff
@@ -29,7 +29,7 @@
     "eqdc",
     "tmerc",
     "stere",
-    "lcca",
```

Comment:
Can I suggest a more descriptive entry is added, so key/description maybe (a dictionary). The acronyms mean absolutely nothing to me, but with more description it will make maintenance more straightforward